pkg/apis/apis.go

```diff
@@ -18,11 +18,15 @@ limitations under the License.
 package apis
 
 import (
+	"github.com/awslabs/operatorpkg/object"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"sigs.k8s.io/karpenter/pkg/apis"
+
+	"github.com/cloudpilot-ai/karpenter-provider-alicloud/config"
 )
 
 var (
 	Group              = "karpenter.k8s.alicloud"
 	CompatibilityGroup = "compatibility." + Group
-	CRDs               = apis.CRDs // object.Unmarshal[apiextensionsv1.CustomResourceDefinition](crds.ECSNodeClassCRD)
+	CRDs               = append(apis.CRDs, object.Unmarshal[apiextensionsv1.CustomResourceDefinition](config.ECSNodeClassCRD))
```

Comment:
cc @helen-frank , I remeber we decide to not use this, do you remeber the reason?

Reply:
The test work needed this, but we didn't have time to write test cases before

Reply:
Got

Reply:
This is used to check the validation rule, I think we can launch a kind to and apply the crd to check whether it's validate.

So, check it in the ut is a little bit redundant and needs to write much code.

What do you think? @daimaxiaxie

Reply:
If e2e, you need to use a lot of CR yaml to verify the rules. I think it is not as convenient as code.